Userland/Applications/SoundPlayer/PlaybackManager.cpp

```diff
@@ -114,10 +114,16 @@ void PlaybackManager::next_buffer()
             return;
         }
 
-        // FIXME: This should handle parsing failures gracefully and show them to the user.
-        auto buffer = m_loader->get_more_samples(m_samples_to_load_per_buffer).release_value();
+        auto buffer_or_error = m_loader->get_more_samples(m_samples_to_load_per_buffer);
+        if (buffer_or_error.is_error()) {
+            // FIXME: These errors should be shown to the user instead of being logged and then ignored
+            warnln("Error while loading samples: {}", buffer_or_error.error().description);
```

Comment:
Can you make this a `dbgln` so it's visible on the debug console instead? Since `SoundPlayer` is a GUI app, most people will not see the `warnln` output.